tests/Tests/600__ai_apps/640__rhoam/640__rhoam_installation.robot

```diff
@@ -0,0 +1,84 @@
+*** Settings ***
+Resource        ../../../Resources/Page/LoginPage.robot
+Resource        ../../../Resources/Common.robot
+Resource        ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../../Resources/Page/ODH/AiApps/Rhoam.resource
+Resource        ../../../../tasks/Resources/RHODS_OLM/uninstall/uninstall.robot
+Resource        ../../../../tasks/Resources/RHODS_OLM/uninstall/oc_uninstall.robot
+Resource        ../../../../tasks/Resources/RHODS_OLM/config/cluster.robot
+Library         ../../../../libs/Helpers.py
+Library         SeleniumLibrary
+Library         OpenShiftLibrary
+Suite Setup     RHOAM Suite Setup
+Suite Teardown  RHOAM Suite Teardown
+
+
+*** Test Cases ***
+Verify RHOAM Can Be Installed
+    [Documentation]    Verifies RHOAM Addon can be successfully installed
+    [Tags]  Tier3
+    ...     ODS-273
+    ${cluster_id}=   Get Cluster ID
+    ${cluster_name}=   Get Cluster Name     cluster_identifier=${cluster_id}
+    Install Rhoam Addon    cluster_name=${cluster_name}
+    Wait Until RHOAM Installation Is Completed    retries=35   retries_interval=2min
+    Verify RHOAM Is Enabled IN RHODS Dashboard
+
+Verify RHODS Can Be Uninstalled When RHOAM Is Installed
```

Comment:
This test is very disruptive because it will leave the cluster without RHODS add-on. I think rhods should be installed back in the test teardown

As this test might take a long time, could you add the tag `Execution-Time-Over-2h`  (or whatever it takes to install all back) ?

Reply:
about RHODS installation, I know already talked with Arthy. I think for now this test should run alone and we can think a better way to re-install RHODS. The problem is time-related.

Sure I can add that tag

Reply:
In tag case (new joke), what do you think about adding a **_Disruptive_** tag to this test? Because you cannot run anything else after it ...